### PR TITLE
Task 63: Llm research

### PR DESCRIPTION
**Pull request by LarisaNichitoaia**

### CODE REVIEW

File: docs/documents/LLM-models.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+Claude 3 model family is the next generation of Anthropic's LLMs. This family includes three state-of-the-art models in ascending order of capability: Claude 3 Haiku, Claude 3 Sonnet, and Claude 3 Opus.
+
+- Pricing
+
+Claude 3/ Claude 2.1 context window - 200,000 token
+Claude 2.0 / Claude Instant context window - 100,000 token
+
+| Claude models   | Input (Per 1M tokens) | Output (Per 1M tokens) |
+| --------------- | --------------------- | ---------------------- |
+| Claude 3 Opius  | $15                   | $75                    |
```
Comment (antonio-marasescu-duran) [inline docs/documents/LLM-models.md]: **suggestion:** correct this to `Claude 3 Opus`

File: docs/documents/LLM-models.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
+# Breef Introduction
```
Comment (antonio-marasescu-duran) [inline docs/documents/LLM-models.md]: **suggestion:** correct this to `Brief`

File: docs/documents/LLM-models.md (hunk 1 above)
Comment (LarisaNichitoaia) [reply docs/documents/LLM-models.md]: done

File: docs/documents/LLM-models.md (hunk 2 above)
Comment (LarisaNichitoaia) [reply docs/documents/LLM-models.md]: done

**Review by antonio-marasescu-duran — APPROVED** [review]

**Pull request merged**